config/routes.rb

```diff
@@ -361,13 +361,22 @@ def external_redirect(path_prefix, target)
           resources :role_appointments, only: %i[new create edit update destroy], shallow: true
           resources :translations, controller: "role_translations"
         end
-        resources :world_locations, only: %i[index edit update show] do
+        # resources :world_locations, only: %i[index edit update show] do
+        #   member do
+        #     get "/features(.:locale)", as: "features", to: "world_locations#features", constraints: { locale: valid_locales_regex }
```

Comment:
suspect this comment should be removed

Reply:
Ah, yes, it should've been.  I'll remove it now.